.travis.yml

```diff
@@ -15,4 +15,5 @@ before_install:
   - sudo apt-get install -qq nodejs
   - echo "deb http://download.mono-project.com/repo/debian wheezy/snapshots/3.12.1 main" | sudo tee /etc/apt/sources.list.d/mono-xamarin.list
   - sudo apt-get install -qq mono-complete
-  - curl -s https://storage.googleapis.com/golang/go1.6.2.linux-amd64.tar.gz | sudo tar -C /usr/local -xzf -
+  - eval "$(sudo gimme 1.6.2)"
```

Comment:
The image is Ubuntu Precise and as far as I can tell it does not have any custom apt package repositories for go. That means the only go package available via apt is some old gcc-go. The gimme script puts out some env variables that are then set in the build environment by `eval`. I don't know which env variables that are, I've just cargo-culted this from the docs. I suppose it sets `$GOROOT` and `$PATH`? If I understand the source code of gimme correctly, go 1.6.2 and a bunch of other go versions is already installed/cached in the build image and gimme just sets the environment variables to point to it. If the go version is not cached gimme falls back to basically what I did with curl.

All default go build images use gimme. I suppose this is best practive[tm]. I didn't know that gimme is available in all travis images, not just the one for `language: go`. (Maybe there is just one image and `language` does something else.)